packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -475,19 +475,23 @@ $expand-less-icon: map-merge(
 }
 
 // Safari-only fix that forces a vertical scrollbar when mobile menu is open.
-// Only needed in the 15px immediately preceding $theme-header-max-width.
+// Only needed in the 15px immediately preceding $theme-header-min-width.
 //
 // Note: 15px is the current width of the Safari scrollbar.
 // More details in https://github.com/uswds/uswds/pull/5443
-$safari-header-bug-min-width: calc(
-  units($theme-header-max-width) - px-to-rem(15px)
-);
-
-@media (min-width: $safari-header-bug-min-width) {
-  .usa-js-mobile-nav--active.is-safari {
-    overflow-y: scroll;
-    position: fixed;
-    // --scrolltop set with JS with zero as fallback.
-    top: var(--scrolltop, 0);
+@if $theme-header-min-width != "none" {
```

Comment:
## Special Tokens

I'm getting a build error when setting `$theme-header-min-width: "auto"`

```js
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
Undefined operation "calc(auto - 0.94rem) > 0".
    ╷
487 │   @if $safari-header-bug-min-width > 0 {
    │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ╵
  packages/usa-nav/src/styles/_usa-nav.scss 487:7  @forward
  packages/usa-nav/src/styles/_index.scss 4:1      @forward
  packages/uswds/_index.scss 38:1                  root stylesheet
  ```
  
- This is a regression from `develop`.
- But it should be noted that setting `auto` breaks the layout of the header.
- Would the error be preferred? If so, should we update the acceptable tokens to exclude values that break the layout? (negative values, 0,) 
- Potentially out of scope

> [!IMPORTANT]
> I began experiencing inconsistent builds and build errors during my testing. There is a chance my broken layouts for certain special tokens _might_ have been a local issue.

Reply:
@mahoneycm I updated the Safari fix to only run if the value of `$theme-header-min-width` fits a _realistic_ value for the setting. 

The header visually breaks and causes warnings when the value of `$theme-header-min-width` is outside of the defined breakpoints in our system (card, card-lg, mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, widescreen). This is because the setting is used primary with our `at-media` mixin, which only accepts the breakpoints defined in `$system-properties`.

For this reason, I decided to bypass the Safari fix unless the setting was defined with the same values that are acceptable to `at-media`.

> [!WARNING]
> There is some risk with this approach. If we ever update the accepted values for the `at-media` mixin to be something other than the system breakpoints, the Safari fix might not be applied in all the right places. However, I feel like the breakpoints in $system-properties will be a stable and reliable place to look for realistic header breakpoints. In my mind, this reduces the risk of future issues.

<details><summary>More on the thought process</summary>
<p>
<ol>
<li>First I tried bypassing only the "special" values of “auto” and “none”. This worked in most cases but would cause an error with 1px/2px values because of the unit mismatch. </li>
<li>This caused me to think about how it would be better to only apply the safari fix to *realistic* values, rather than *possible* values. The header visually breaks and causes warnings when the value of `$theme-header-min-width` is outside of the defined breakpoints in our system (card, card-lg, mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, widescreen). For this reason, I decided to bypass the Safari fix unless the setting was defined with one of those values. </li>
<ul>
<li>Note: there is some risk with this because if we ever update the accepted values for the `at-media` mixin, the Safari fix might not be applied in all the right places. However, I feel like the breakpoints in $system-properties will be a stable and reliable place to look, which reduces the risk of future issues in my mind. 
</ul>
<li> Alternatively, we could create a custom map that includes all possible reasonable values, which could mean including the "medium" spacing token set. I ultimately moved away from this because it seemed very rare that a mobile nav would be viewed in safari <120px, especially if the header is broken in other ways by this value.  </li>
</ol>
</p>
</details> 

Can you confirm the following?
- Setting `$theme-header-min-width` to any breakpoint value (card, card-lg, mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, widescreen)
  -  does NOT cause a compile error 
  -  DOES generate an `is-safari` class in the CSS 
- Setting `$theme-header-min-width` to any non-breakpoint value (Ex: "none", "auto", 15, "hamburger")
  -  does NOT generate an `is-safari` class in the CSS 
  -  does NOT cause a compile error. However, this warning is still expected for non-breakpoint values: 
```
Warning: `none` is not a valid USWDS project breakpoint. Valid values: card, card-lg, mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, widescreen
```

Reply:
I'm still seeing `is-safari`. The logic looks good and the build warning are appearing, but the safari class is added. I've tried rebuilding the project multiple times.

The confusing thing is when I run a debug, it returns `false` as expected and should _not_ be running the block of scss 🤔 

```scss
@debug map-has-key($our-breakpoints, $theme-header-min-width); // Output: false
```

- [x]  Standard breakpoint generates `is-safari` class
- [x]  No compilation errors
- [ ]  Non-breakpoint values bypass safari fix
- [x]  Non-breakpoint values cause compilation warning, but no error

Is the `is-safari` class being omitted for you?